src/utils/test.ts

```diff
@@ -2,8 +2,53 @@ import { expect } from "chai";
 import getHyperLinkFromPath from "./getHyperLinkFromPath";
 
 describe("getHyperLinkFromPath Function", () => {
-  it("Should change the path to github link", () => {
+  it("Should return [reusable](./src/components/reusable)", () => {
     const result = getHyperLinkFromPath("src/components/reusable");
     expect(result).to.equal("[reusable](./src/components/reusable)");
   });
 });
+
+describe("getHyperLinkFromPath Function", () => {
+  it("Should return [reusable](./src/components/reusable)", () => {
+    const result = getHyperLinkFromPath("src");
+    expect(result).to.equal("[src](./src)");
+  });
+});
+
+describe("getHyperLinkFromPath Function", () => {
+  it("should return Error", () => {
+    expect(() => getHyperLinkFromPath("./src")).to.throw();
+  });
+});
+describe("getHyperLinkFromPath Function", () => {
+  it("should return Error", () => {
+    expect(() => getHyperLinkFromPath("./src/")).to.throw();
+  });
```

Comment:
@sahandfarahani @Manpreet-Bhatti This is not how we structure tests with the same name describes / multiple describes?